nox/sessions.py

```diff
@@ -242,8 +242,11 @@ def install(self, *args, **kwargs):
 
         if "silent" not in kwargs:
             kwargs["silent"] = True
+        upgrade = kwargs.pop("upgrade", True)
```

Comment:
I am all for this but I'd make `False` the default

Reply:
That's a backwards incompatible change, but I'm cool with if you're cool with it too.